eslint.config.mjs

```diff
@@ -0,0 +1,9 @@
+import pluginJs from '@eslint/js';
+import eslintConfigPrettier from 'eslint-config-prettier';
+import globals from 'globals';
+
+export default [
+  { languageOptions: { globals: globals.browser } },
+  pluginJs.configs.recommended,
```

Comment:
```suggestion
  // See https://eslint.org/docs/latest/rules/ to understand theses rules
  pluginJs.configs.recommended,
```

What do you thinks about having link to docs ?

Reply:
cc @araujogui